### PR TITLE
V0.3.1 rc

### PR DESCRIPTION
**Pull request by gzagaris**
This PR starts the v0.3.1 release candidate for Axom.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -122,7 +232,8 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 -
 
 
-[Unreleased]: https://lc.llnl.gov/bitbucket/projects/ATK/repos/axom/compare/commits?targetBranch=refs%2Ftags%2Fv0.2.9&sourceBranch=refs%2Fheads%2Fdevelop&targetRepoId=1066
-[Version 0.2.9]: https://lc.llnl.gov/bitbucket/projects/ATK/repos/axom/compare/commits?targetBranch=refs%2Ftags%2Fv0.2.8&sourceBranch=refs%2Ftags%2Fv0.2.9&targetRepoId=1066
+[Unreleased]:    https://github.com/LLNL/axom/compare/v0.3.0...develop
```
Comment (kennyweiss) [inline RELEASE-NOTES.md]: Thanks @gzagaris.

Could you please add/update these links as well?
```
 [Unreleased]:   https://github.com/LLNL/axom/compare/v0.3.1...develop 
 [Version 0.3.1]: https://github.com/LLNL/axom/compare/v0.3.0...v0.3.1
 [Version 0.3.0]: https://github.com/LLNL/axom/compare/v0.2.9...v0.3.0 
```

Comment (gzagaris) [reply RELEASE-NOTES.md]: Fixed. Thanks.

File: CODE-OF-CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at blt-dev@llnl.gov. All
```
Comment (ltaylor16) [inline CODE-OF-CONDUCT.md]: Update address from blt-dev to axom-dev.

Comment (gzagaris) [reply CODE-OF-CONDUCT.md]: Good catch! Thanks.

**Pull request merged**